apps/docs/content/components/Image.mdx

```diff
@@ -0,0 +1,90 @@
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Image src="https://source.unsplash.com/random" alt="Random Image" />
+```
+
+### Picture
+
+The `as` prop can be used to set the container of the image to be a picture component.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  as="picture"
+/>
+```
+
+### Aspect ratio
+
+The `aspectRatio` prop can be used to set the aspect ratio of the image. This is useful when the image is not the same aspect ratio as the container.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  aspectRatio="16:9"
+/>
+```
+
+### Height
+
+The `height` prop can be used to set the height of the image. This can be used along side the `aspectRatio` prop to create a responsive image the same size as other images.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  height={200}
+  aspectRatio="16:9"
+/>
+```
+
+### Width
+
+The `width` prop can be used to set the width of the image. This can be used along side the `aspectRatio` prop to create a responsive image the same size as other images.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  width={200}
+  aspectRatio="16:9"
+/>
+```
+
+## Component props
+
+### Image <Label>Required</Label>
+
+The first row in a table is automatically styled as a table header row.
+
+| name          | type                                    | default     | required | description                                                                                                                                                               |
+| ------------- | --------------------------------------- | ----------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `src`         | `string`                                |             | `true`   | Specifies the path to the image                                                                                                                                           |
+| `alt`         | `string`                                |             | `true`   | Specifies an alternate text for the image                                                                                                                                 |
+| `as`          | `img`, `picture`                        | `img`       | `false`  | Specification to create a picture component                                                                                                                               |
+| `aspectRatio` | `1:1`, `16:9`, `16:10`, `4:3`, `custom` | `undefined` | `false`  | Sets the image aspect ratio. A custom ratio can be provided in the design tokens.                                                                                         |
```

Comment:
Is `custom` still a valid value, or were we dropping it. If it's valid, can we add an example of how users should use it please?

Reply:
Could we add quotes around these to convey these are strings?

```suggestion
| `aspectRatio` | `'1:1'`, `'16:9'`, `'16:10'`, `'4:3'`, `'custom'` | `undefined` | `false`  | Sets the image aspect ratio. A custom ratio can be provided in the design tokens.                                                                                         |

````